localMatchApp/headers/Login.h

```diff
@@ -0,0 +1,44 @@
+/**
+ * \class Login
+ *
+ * \brief Create the connection with the database and the core.
+ *
+ * This class check out the format of the password and check if the
+ * user exists, in the sign-up function.
+ *
+ *
+ */
+
+#ifndef LOGIN_H
+#define LOGIN_H
+
+#include <string>
+class Login {
+ public:
+  /// Create a class type enum to communicate the state of the methods "signUp" and "entry".
```

Comment:
Change comment. First part is unnecessary "Create a class type enum to communicate..."

Reply:
Done.